web/source/kmwcallback.js

```diff
@@ -113,6 +113,9 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
 
       // Execute any external (UI) code needed after loading keyboard
       keymanweb.doKeyboardLoaded(Pk['KI']);
+
+      // Restore the originally-active stub to its prior state.  No need to change it permanently.
+      keymanweb._ActiveStub = as;
```

Comment:
It's quite possible that we could outright remove the `keymanweb._ActiveStub` stuff from this method, but just in case this behavior was actually used by existing API, doing a small wrapper that resets the value after event generation should help minimize impact while fixing the buggy behavior.

This allows a keyboard to be registered without overriding the currently-set keyboard and is the other half to the fix for the help.keyman.com example, allowing any default-behavior keyboard loading to not override post-`init()` user or API settings for the active keyboard; without the fix, it is impossible to set the 'default' keyboard as active until the KMW-default keyboard's KR call completes asynchronously.

Reply:
Ok; can we rename `as` to `savedActiveStub` or something like that?

Reply:
Sure, I'll get right on that.